spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMetadataTables.java

```diff
@@ -530,12 +530,13 @@ public void testFilesTableTimeTravelWithSchemaEvolution() throws Exception {
                 + tableName
                 + ".files VERSION AS OF "
                 + currentSnapshotId
-                + " ORDER BY content");
+                + " ORDER BY file_path");
     List<Row> actualFiles = TestHelpers.selectNonDerived(actualFilesDs).collectAsList();
     Schema entriesTableSchema = Spark3Util.loadIcebergTable(spark, tableName + ".entries").schema();
     List<ManifestFile> expectedDataManifests = TestHelpers.dataManifests(table);
     List<Record> expectedFiles =
         expectedEntries(table, FileContent.DATA, entriesTableSchema, expectedDataManifests, null);
+    expectedFiles.sort(Comparator.comparing(r -> r.get("file_path").toString()));
```

Comment:
Ordering by `file_path` for both actual files and expected files sorts both list deterministically.